### ISSUE TITLE
Plugin not toggling relative linenumbers (Linux, 0.169.0)

### ISSUE THREAD

**Issue by LinuCC** [ridderholt/atom-relative-linenumbers#5]
The Plugin seems to do nothing when trying to toggle the relative linenumbers.
The absolute numbers are always displayed.
Standard Shortcut, custom shortcut and Button "Relative Lines->Toggle" in Menu Bar were tried, including reinstall of plugin.
Is this just for me?
(New Atom-User, never had it working before.)

**Comment by geoffharcourt** [comment]
I don't think this issue is Linux specific, I'm seeing it on OSX build (0.177.0).

**Comment by ghost** [comment]
Hi and thanks for reporting this issue, I'm aware of the problem which is caused by that I haven't had the time to keep the plugin up to date with the API changes the Atom team has made. I will try to fix this but I can't promise anything. I'm will happily accept any pull request which solves this issue.

**Comment by Nadrieril** [comment]
I believe the proper way to do this would be to change the atom GutterComponent instead of modifying the DOM when we receive events... It would be easier if it was an independant package

**Comment by Nadrieril** [comment]
I made a pull request to atom core: https://github.com/atom/atom/pull/5449
